Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM ffig/ffig-base
 MAINTAINER FFIG <support@ffig.org>
 
 COPY . /home/ffig
+RUN find /home/ffig -type f -exec dos2unix {} \;
```

Comment:
At the very least, use `-exec dos2unix {} +` (see primary comment in Conversation tab).